src/content/reference/gsctl/update-cluster.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "gsctl Command Reference: update cluster"
+description: The 'gsctl update cluster' command allows the modification of the name of a cluster and its labels.
+date: 2020-05-04
+type: page
+weight: 44
+---
+
+# `gsctl update cluster`
+
+The `gsctl update cluster` allows the modification of the name of a cluster and its labels.
+Cluster labelling is only available for clusters with node pools on AWS.
+
+## Usage
+
+The command is called with the cluster ID or name as a positional argument.
+The desired new name can be specified with the `--name` or `-n` flag.
+The `--label` flag is used to modify labels.
+It can receive either a JSON formatted [JSON Merge Patch, RFC 7386](https://tools.ietf.org/html/rfc7386) or a string specifying label changes.
+
+### Name modification example
+
+```nohighlight
+$ gsctl update cluster f01r4 --name "Precious Production Cluster"
+```
+
+### Labels modification JSON Merge Patch example
+
+```nohighlight
+$ gsctl update cluster vxvc7 --labels '{"environment": "testing", "locked": null}'
```

Comment:
Not digging the syntax (use of JSON) here. Let's come up with something that is more CLI-native.

Reply:
so only the `key=value` syntax then?

Reply:
And then I found the syntax described below. When we make that usable, I see no reason to support JSON in the CLI.

Reply:
What about people using gsctl in automation not wanting to talk directly to the API? I am very much fine with implementing less, but the original story (https://github.com/giantswarm/giantswarm/issues/1693) even mentions usage of tenant cluster labels in CI context

Reply:
> What about people using gsctl in automation not wanting to talk directly to the API?

They should be able to use `--label` one or several times in the non-JSON way.

Reply:
We'll do it the multiple `--label` way to get implementation tires kicking..